packages/http/httpcall_client.js

```diff
@@ -173,6 +173,19 @@ HTTP.call = function(method, url, options, callback) {
       }
     };
 
+    // Allow custom control over XHR and abort early.
+    if(options.beforeSend) {
+      // Sanity
+      var beforeSend = _.once(options.beforeSend);
+
+      // Possibly change the call so the xhr is the context.
```

Comment:
This was just a note to self, to possibly discuss weather the context of the callback should be the instance of the XHR over the XHR being an argument, but as you stated, Meteor is moving away from this.* in favour of arguments.